libsplinter/src/service/scabbard/client/submit.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Contains functions which assist with batch submission to a REST API
+
+use std::{fmt, str};
+
+use protobuf::Message;
+use reqwest::{
+    blocking::{Client, RequestBuilder, Response},
+    Url,
+};
+
+use sawtooth_sdk::messages::batch::BatchList;
+
+use crate::service::scabbard::{BatchInfo, BatchStatus};
+
+use super::Error;
+
+const BATCHES_ENDPOINT: &str = "/batches";
+const HTTP_SCHEME: &str = "http";
```

Comment:
Both of the above const make the code less clear than if the strings were used directly.

Reply:
Updated